tests/Models/Solo/ScoreTest.php

```diff
@@ -21,6 +22,62 @@ public function testCreateLegacyEntryIncompletePlay()
         $score->createLegacyEntry();
     }
 
+    public function testLegacyPassScoreRetainsRank()
+    {
+        $score = new Score([
+            'mods' => [],
+            'user_id' => 1,
+            'beatmap_id' => 1,
+            'ruleset_id' => 1,
+        ]);
+
+        $score->complete([
+            'ended_at' => Carbon::now(),
+            'passed' => true,
+            'total_score' => 1000,
+            'max_combo' => 100,
+            'statistics' => ['great' => 10],
```

Comment:
doesn't really affect anything but I think the client sends `Great`, not `great`